protocol/avdl/keybase1/home.avdl

```diff
@@ -89,6 +94,15 @@ protocol home {
     default: void;
   }
 
+  variant HomeScreenTodoExt switch (HomeScreenTodoType t) {
```

Comment:
@heronhaye another look if this is necessary to extend existing todo? I'm not a big fan of having to do all this code on both server and client becasue of avdl limitation

Reply:
what's wrong with the data you already added?

Reply:
We need to somehow communicate to the client when was the last time we sent the verification email. I'd imagine another way to approach this would be to communicate this through gregor state?

Reply:
I think there are two separate issues here:
1) returning extra data from `/home` handler. We can't change existing `HomeScreenTodo` variant without making it incompatible with client going out today. So one way is adding separate field entirely, which is not ideal, especially considering it will be used just for one todo type.
2) Refreshing people tab when a button is clicked. I think because this additional field does not increase todo version, clients beside the one that makes that action will not know to refresh that data. But I'm not sure if that's exactly how it works.